torch/utils/_sympy/value_ranges.py

```diff
@@ -443,6 +443,10 @@ def floor(cls, x):
     def ceil(cls, x):
         return ValueRanges.increasing_map(x, sympy.functions.elementary.integers.ceiling)
 
+    @classmethod
+    def round(cls, number, ndigits=None):
+        return ValueRanges(sympy.floor(number.lower), sympy.ceiling(number.upper))
```

Comment:
I think this is wrong for negative `ndigits` as we can round up or down by more than 1.